CONTRIBUTING.md

```diff
@@ -390,38 +391,40 @@ A good metric:
-`gatsby-config.js`](https://github.com/newrelic/developer-website/blob/ae42737f5f1cf556f3c44d864655c9a571739e28/gatsby-config.js#L161)
-with the new release number to update.
+- We use a [local Gatsby plugin `gatsby-source-newrelic-sdk`](https://github.com/newrelic/developer-website/tree/develop/plugins/gatsby-source-newrelic-sdk)
+to source our documentation into GraphQL. This plugin has some configuration that tells it what release number to use.
 
-To obtain the release number visit the `wanda-ec-ui` repository on GH enterprise and
-look at the release version.
+- [Update `gatsby-config.js`](https://github.com/newrelic/developer-website/blob/ae42737f5f1cf556f3c44d864655c9a571739e28/gatsby-config.js#L161)
+with the new release number to update the bundle release version.
+
+- To obtain the version release number visit the `wanda-ec-ui` repository on Github enterprise and look at the release version.
 
 ### Step 2: Add any new APIs or components to our constants list
 
-At the time of this writing, we rely on the 1st party documentation bundle to
-power the developer docs. While the 1st party bundle provides many of the same
-components/APIs, there are a few minor differences between the 1st and 3rd party
-SDKs. To account for this, we list out the components we document on the site.
-If there are any new components or APIs, [update the constants
+- We rely on the 1st party documentation bundle to power the developer docs. While the 1st party bundle provides many of the same components/APIs, there are a few minor differences between the 1st and 3rd party SDKs. To account for this, we white list the specific components we document on the site.
+- If there are any new components or APIs, [update the constants
 list](https://github.com/newrelic/developer-website/blob/develop/plugins/gatsby-source-newrelic-sdk/src/constants.js)
-with the new components. Pages will be automatically generated for each of
-these.
+with the new components.
+- Pages will be then automatically generated for each of these.
 
-Once we get a 3rd party bundle built for us, we should no longer need this step
-as that will contain only 3rd party SDK documentation.
+> Once we have a 3rd party bundle automatically built for us, we should no longer need this step as that will contain only 3rd party SDK documentation. That is an open request to the NR One Core Team.
 
 ### Step 3: Add any new APIs or components to the navigation
 
 If there are new APIs or components, we will want to list them in the navigation
 so that a user can easily discover them. [Add an entry to `nav.yml`](https://github.com/newrelic/developer-website/blob/develop/src/data/nav.yml)
 to get the new API/component in the nav.
+
+### Step 4: Misc Info
+
+- This [file](https://github.com/newrelic/developer-website/blob/develop/plugins/gatsby-source-newrelic-sdk/src/navigationApi.js) contains the manually generated navigation API this is currently missing from the API.
```

Comment:
I'd hesitate to include all of this info here, mostly because the individual files may have a lot of churn and a lot of this stuff is internal implementation details. Maintaining the docs each time we make a change could get tiresome.

For example, this particular line about the navigation API is only useful while we need to polyfill that value. As soon as the platform provides documentation for it, we will then be able to go delete this.

I'll leave it up to you to decide whether its worth it, but just know this might be a bit volatile and could get out of date super quickly.